src/pallets_sphinx_themes/__init__.py

```diff
@@ -5,7 +5,11 @@
 import textwrap
 from collections import namedtuple
 
-import importlib_metadata
+if sys.version_info >= (3, 8):
```

Comment:
Use `except ImportError` instead of version checks.

Reply:
Changed and rebased.

Reply:
Also thanks for the super fast review :)